.jshintrc

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Seems like we might be making a shift to ESHint. Can you check in with @amb26 to see if this should be switched over.